### PR TITLE
web readme t2 config

### PR DESCRIPTION
**Pull request by RAY-316**
https://github.com/Azure/sdk-release-request/issues/1419

### CODE REVIEW

**Review by weidongxu-microsoft — APPROVED** [review]

File: specification/web/resource-manager/readme.md [hunk 1]
```diff
@@ -1263,6 +1263,10 @@ swagger-to-sdk:
 
 See configuration in [readme.go.md](./readme.go.md)
 
+## Python
+
+See configuration in [readme.python.md](./readme.python.md)
+
```
Comment (msyyc) [inline specification/web/resource-manager/readme.md:1269]: Is '2020-12-01' added to 'readme.python.md' ?

**Comment by miwebst** [comment]
@RAY-316 / @msyyc before releasing this, we wanted to get this breaking change in. It was also blocked on the python auto-rest issue. Can you please retrigger the python job there and merge the PR before generating the sdk?

https://github.com/Azure/azure-rest-api-specs/pull/13768

**Review by 00Kai0 — APPROVED** [review]

**Comment by msyyc** [comment]
For python, the breaking change is ok

**Review by msyyc — APPROVED** [review]

**Comment by msyyc** [comment]
@ArcturusZhang Please help on the failure of GO.

**Pull request merged**